modules/exploits/linux/http/netgear_r7000_cgibin_exec.rb

```diff
@@ -40,7 +40,8 @@ def initialize(info = {})
           'Space'       => 1024,
           'DisableNops' => true,
           'BadChars'    => "\x20"
-        }
+        },
+      'DefaultOptions'  => { 'WfsDelay' => 10}
```

Comment:
This wasn't necessary in my testing. What happened in your case? I was able to get a shell immediately.